data/language/pl-PL.yml

```diff
@@ -1580,31 +1567,31 @@ strings:
   1538: '{SMALLFONT}{COLOUR BLACK}Następny utwór'
   1539: Odtwarzaj utwory na bieżącą epokę
   1540: Odtwarzaj wszystkie utwory
-  1541: Odtwarzaj indywidualnie dobrane utwory
-  1542: Edytuj wybór...
-  1543: '{SMALLFONT}{COLOUR BLACK}Edit the music track selection list'
-  1544: Edytuj Wybór Muzyki
+  1541: Odtwarzaj wybrane utwory
+  1542: Edytuj Listę Utworów...
+  1543: '{SMALLFONT}{COLOUR BLACK}Edyduj listę odtwarzanych utworów'
+  1544: Lista Utworów Muzycznych
   1545: '{SMALLFONT}{COLOUR BLACK}Kliknij na utwór muzyczny, aby przełączyć wybór'
   1546: ✓
   1547: '{COLOUR WINDOW_2}Głośność:'
   1548: '{SMALLFONT}{COLOUR BLACK}Ustaw głośność muzyki'
   1549: Opcje Muzyki
-  1550: Wybierz twarz właściciela dla {STRINGID}
-  1551: '{SMALLFONT}{COLOUR BLACK}Kliknij, aby wybrać twarz firmy/właściciela'
+  1550: Wybór Twarzy Właściciela Firmy {STRINGID}
+  1551: '{SMALLFONT}{COLOUR BLACK}Kliknij, aby wybrać twarz firmy / właściciela'
   1552: Już wybrano dla innej firmy
   1553: Nie można wybrać twarzy...
   1554: lotniska
   1555: porty
   1556: "{COLOUR WINDOW_2}Rozpoczęcie działalności: {COLOUR BLACK}{DATE MY}"
-  1557: '{COLOUR WINDOW_2}Wskaźnik Wydajności: {COLOUR BLACK}{INT16_1DP}% {NEWLINE}  “{STRINGID}”'
-  1558: "{COLOUR WINDOW_2}Wskaźnik Wydajności: {COLOUR BLACK}{INT16_1DP}%{SPRITE 2325}{NEWLINE}  “{STRINGID}”"
-  1559: "{COLOUR WINDOW_2}Wskaźnik Wydajności: {COLOUR BLACK}{INT16_1DP}%{SPRITE 2324}{NEWLINE}  “{STRINGID}”"
+  1557: '{COLOUR WINDOW_2}Wskaźnik wydajności: {COLOUR BLACK}{INT16_1DP}%{NEWLINE}{STRINGID}'
```

Comment:
Please leave the quotation marks here, rather than hardcoding them in the argument strings.